lib/Badge.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Material badge for showing notifications, maybe more functionality in the future
+ * @flow
+ */
+
+import React, { Component } from 'react'
+import {
+  View,
+  Text,
+  StyleSheet
+} from 'react-native';
+
+type BProps = {
+  text: string,
+  size: number,
+  children: Array<ReactElement<*>>,
+  style: any,
+  isVisible: boolean,
+}
+
+const defaultProps = {
+  size: 20,
+  isVisible: true,
+  style: {},
+}
+
+export default class RippleBackgroundTransition extends Component {
+  static defaultProps: typeof defaultProps
+  props: BProps
+  static defaultProps = defaultProps
+
+  createStyles() {
+    const { size, style } = this.props;
+    const container = Object.assign(
+      {},
+      {
+        position: "relative",
```

Comment:
_For android compatibility:_ Change to `position: 'absolute'`

Reply:
And to make the `position: absolute` work, add `position: 'relative'` to Tab.js, `StyleSheets.container`. Somewhere here:

https://github.com/ShayanJavadi/react-native-material-bottom-navigation/blob/3f70221a611029b13f416f6ab51c2de569679d40/lib/Tab.js#L295-L304

Reply:
done